src/uwloc/data/main.py

```diff
@@ -1,13 +1,15 @@
 import glob
 import os
+from datetime import datetime, timezone
 
 import typer
 from typing_extensions import Annotated
 
-from .tile import MAX_HOURS, MAX_UNITS, create, init_tiledb, write_row
-from .wav import read_wav
+from .tile import MAX_HOURS, MAX_UNITS, create, init_tiledb, tidy, write_row
+from .wav import read_wav, read_wav_metadata
 
 app = typer.Typer()
+WAV_PATTERN = "**/*.[wW][aA][vV]"
```

Comment:
Global variables should come right after imports. Swap this line with the line above here.

Reply:
They are both global variables ;), but I know what you mean...